sematic/db/models/resolution.py

```diff
@@ -170,6 +167,8 @@ class Resolution(Base, JSONEncodableMixin, HasExternalJobsMixin):
     external_jobs_json: Optional[List[Dict[str, Any]]] = Column(
         types.JSON(), nullable=True
     )
+    container_image_uris: Optional[Dict[str, str]] = Column(types.JSON(), nullable=True)
+    container_image_uri: Optional[str] = Column(types.String(), nullable=True)
```

Comment:
Please rename `container_image_uri` to something like `default_container_image_uri` and please document these via code comments.

Reply:
It's not the default container image URI. It is the container image that is selected for the driver job. It can be default, or another one depending on the `_base_tag_image` value passed.
Adding comments though.